babel-plugin-optimize-obj-str/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "babel-plugin-optimize-objstr",
+  "version": "1.0.0",
+  "repository": "lukeed/obj-str",
+  "description": "Babel plugin to optimize `obj-str` calls.",
+  "main": "src/index.js",
+  "license": "MIT",
+  "author": {
+    "name": "Luke Edwards",
+    "email": "luke.edwards05@gmail.com",
+    "url": "lukeed.com"
+  },
+  "contributors": [
+    {
+      "name": "Alan Orozco",
+      "email": "alan@orozco.xyz",
+      "url": "https://alanoroz.co"
+    }
+  ],
+  "scripts": {
+    "test": "jest --detectOpenHandles"
```

Comment:
This has not been configured on CI, please consider as a possible addition.

Reply:
Given that `obj-str` currently uses `tape` for testing, should this plugin do the same?

Reply:
It seems like `@babel/helper-plugin-test-runner` only works with `jest` unfortunately. But I may be holding it wrong?

Reply:
I'll likely move both suites to uvu – so this is fine for now :)